colour-mapping/README.rst

```diff
@@ -0,0 +1,45 @@
+Colour Mapping Demo
+===================
+A demo on using the D-Wave Ocean SDK to solve the colour mapping problem. Namely,
+given a map, colour the regions of the map such that no two regions sharing a
+border would share the same colour.
+
+.. figure:: graph.png
+  :align: center
+  :figclass: align-center
+  
+  A graph representation of Canada's provinces. Provinces connected by edges share
+  a border together. This is a sample output produced by this demo.
+
+Usage
+-----
+Simply run the code with
+::
+  python colour_mapping.py
+
+Code Overview
+-------------
+The idea is to describe the colour mapping problem in terms of constraints.
+Once this is done, we can use tools from the Ocean SDK to convert these
+constraints into a binary quadratic model (BQM), a type of equation that can be
```

Comment:
Do we want to define BQM as an equation?

Reply:
Since this is just a code overview, I think it's suffice to say that a BQM is just an equation that the quantum computer accepts.